modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/MessagingEngine.java

```diff
@@ -644,6 +654,11 @@ public void close() throws InterruptedException {
             log.warn("Content remover task forcefully shutdown.");
             throw e;
         }
+
+        //Stop Slot manager in coordinator
+        if (AndesContext.getInstance().isClusteringEnabled() && AndesContext.getInstance().getClusteringAgent().isCoordinator()) {
+            ClusterResourceHolder.getInstance().getClusterManager().getSlotManager().shutDownSlotManager();
```

Comment:
As at now, this method will close the SlotDeletionSafeZone calculating worker threads. if needed we can add any similar workers here in future.